nimiq-ruby-client.gemspec

```diff
@@ -0,0 +1,34 @@
+# lib = File.expand_path("lib", __dir__)
+# $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+# require "nimiq/version"
+require_relative "lib/nimiq-client"
```

Comment:
I'm getting the errors below: 

```
$ gem build nimiq-ruby-client.gemspec 
Invalid gemspec in [nimiq-ruby-client.gemspec]: cannot load such file -- oj
ERROR:  Error loading gemspec. Aborting.

$ bundle
[!] There was an error parsing `Gemfile`: 
[!] There was an error while loading `nimiq-ruby-client.gemspec`: cannot load such file -- oj
Does it try to require a relative path? That's been removed in Ruby 1.9. Bundler cannot continue.
```
The errors go away by uncommenting the three lines at the top and commenting the line with the relative require. So that anyone wanting to install the gem from the local folder for development proposes could do it without any issues, do you think those three lines could be uncommented by default in the file and the relative require removed?

This also seems to work, but it's a bit shorter:

```
$:.push File.expand_path("../lib", __FILE__)
require "nimiq/version"
```

Reply:
Did you `bundle` before that? also ruby what version are you using?

Reply:
Somehow the error is gone now, I tried to bundle in multiple occasions before running `gem build nimiq-ruby-client.gemspec` and I was getting always the same errors about `require_relative` being removed in Ruby 1.9. Today I tried yet again and the whole operation completed with no problem, so I'm puzzled. Nevertheless, I'm still intrigued by the error about Ruby dropping support for `require_relative`, I took the snippet I posted above from the `gemspec` file in this [Bitcoin RPC client repository](https://github.com/sinisterchipmunk/bitcoin-client) but to be honest I'm not sure about the [side effects](https://github.com/grpc/grpc/pull/11124) of having `require_relative` vs pushing the directory with the local module into the array of paths and making a "normal" `require`, so feel free to ignore my suggestions in this thread.